assets/src/apps/delivery/layouts/deck/components/FeedbackContainer.tsx

```diff
@@ -70,7 +70,14 @@ const FeedbackContainer: React.FC<FeedbackContainerProps> = ({
             </button>
           </div>
           <style type="text/css" aria-hidden="true" />
```

Comment:
probably could have used this one, not sure why it was here... not sure if aria-hidden is needed for style blocks and screen readers, anyway, we can delete it now